src/NDItk/multigroup/EnergyGroupStructure.hpp

```diff
@@ -38,12 +46,8 @@ class EnergyGroupStructure : protected base::RealListRecord {
     return this->empty() ? 0 : this->size() - 1;
   }
 
-  /**
-   *  @brief Return the group structure boundaries
-   */
-  auto boundaries() const { return this->values(); }
```

Comment:
You don't have this anymore because now it exists in PrimaryStructure and OutgoingStructure?

Reply:
values() is on the base class and I thought adding an actual boundaries() when values() is already available makes no sense.